src/test/util/setup_common.cpp

```diff
@@ -185,12 +185,12 @@ TestingSetup::TestingSetup(const std::string& chainName, const std::vector<const
     assert(!::ChainstateActive().CanFlushToDisk());
     ::ChainstateActive().InitCoinsCache(1 << 23);
     assert(::ChainstateActive().CanFlushToDisk());
-    if (!::ChainstateActive().LoadGenesisBlock(chainparams)) {
+    if (!::ChainstateActive().LoadGenesisBlock()) {
```

Comment:
If you touch this branch again, consider removing the local `chainparams` reference and just calling `Params()` in the call to `PeerManager::make()` below.

Reply:
The line will be touched again in the follow-up that removes `::Params()`, so I'll leave it as is for now.